src/main/lib/ccd-spreadsheet-utils.js

```diff
@@ -75,4 +67,33 @@ class SpreadsheetBuilder {
   }
 }
 
-module.exports = { SpreadsheetBuilder, SpreadsheetConvert };
+class JsonHelper {
+  static dateFieldToString(fieldname, json) {
+    json.forEach(obj => {
+      if (obj[fieldname]){
+        obj[fieldname] = XlsxPopulate.numberToDate(obj[fieldname]).toLocaleDateString();
+      }
+    });
+  }
+
+  static stringToDateField(fieldname, json){
+    json.forEach(obj => {
+      if (obj[fieldname]){
+        let dateString = obj[fieldname];
+        obj[fieldname] = XlsxPopulate.dateToNumber( new Date(dateString));
```

Comment:
Since `Date` API is one of the weak points in JS let's use `moment` library for date processing. Moment became JS standard date processing library. 

See: https://momentjs.com